scripts/01a_create_data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source /home/ubuntu/.pyenv/versions/venv.cnet/bin/activate
+
+PROJECT_PATH="/home/ubuntu/cultionet_train"
+CROP_COLUMN=""
+REPLACE_DICT=""
+
+cultionet create -p $PROJECT_PATH -gs 100 100 --crop-column $CROP_COLUMN \
```

Comment:
ok! So there are few added scripts. I assume this is to streamline the process of creating, predicting, etc...?

Reply:
Correct, and to provide CLI examples.